$OpenBSD$
Use gflex instead of flex from base system, or things won't work.
--- qtwebkit/Tools/qmake/mkspecs/features/default_pre.prf.ports.orig	Thu Nov  6 12:54:13 2014
+++ qtwebkit/Tools/qmake/mkspecs/features/default_pre.prf	Thu Nov  6 12:55:52 2014
@@ -105,7 +105,11 @@ if(win32|mac):!macx-xcode {
 
 # A newer version of flex is required on Windows. At the moment the only
 # one that appears to provide binaries and is not cygwin is winflex.
+#
+# On OpenBSD, you'll need newer flex from ports/packages, named gflex.
+# The flex program from base system won't satisfy our needs.
 FLEX = flex
+openbsd: FLEX = gflex
 win32: FLEX = win_flex
 
 BIN_EXTENSION =
